pkg/client/client.go

```diff
@@ -1862,6 +1862,14 @@ func (c *Client) PodCapacity(ctx context.Context) (int, error) {
 	return int(podCapacityTotal), nil
 }
 
+func (c *Client) HasConsoleOperatorEnabled(ctx context.Context) (bool, error) {
+	_, err := c.oscclient.ConfigV1().ClusterOperators().Get(ctx, "console", metav1.GetOptions{})
```

Comment:
(suggesetion) I'd rather check the status of the clusterversion/cluster resource which should tell us if the `Console` capability is enabled or not.

https://github.com/openshift/api/blob/ea40115b9fa67188f89075ec4e34a9ea346ea123/config/v1/types_cluster_version.go#L441-L454

https://github.com/openshift/api/blob/ea40115b9fa67188f89075ec4e34a9ea346ea123/config/v1/types_cluster_version.go#L271-L273

Reply:
Thanks @simonpasquier, Nice find. I will modify the code rely on this instead.